src/BeautifulComments/ClyPackageRichTextCommentEditorToolMorph.class.st

```diff
@@ -45,10 +45,9 @@ ClyPackageRichTextCommentEditorToolMorph class >> tabOrder [
 { #category : 'operations' }
 ClyPackageRichTextCommentEditorToolMorph >> applyChanges [
 
-	self pendingText asString = self editingText ifFalse: [
-		editingPackage packageManifest comment: self pendingText stamp: Author changeStamp].
+	self pendingText asString = self editingText ifFalse: [ editingPackage packageManifest comment: self pendingText stamp: (Date today mmddyyyy, ' ' , ((String streamContents: [:s | Time now print24: true on: s]) copyFrom: 1 to: 5)) ].
```

Comment:
I would put timestamp computation in a new method to increase readability and also use [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) yyyymmdd format instead of mmddyyyy.
```smalltalk
editionTimestamp
  ^ Date today yyyymmdd, ' ', (Time now print24 first: 5).

applyChanges
  self pendingText asString = self editingText 
     ifFalse: [ editingPackage packageManifest comment: self pendingText stamp: self editionTimestamp ]
```